cocktails.mcp/src/.env.test

```
@@ -1,9 +1,8 @@
 APPLICATIONINSIGHTS_INSTRUMENTATIONKEY=00000000-0000-0000-0000-000000000000
-AZUREAD_B2C_CLIENT_ID=00000000-0000-0000-0000-000000000000
-AZUREAD_B2C_DOMAIN=cezzis.onmicrosoft.com
-AZUREAD_B2C_INSTANCE=https://testlogin.cezzis.com
-AZUREAD_B2C_USERFLOW=B2C_1_SignInSignUp_Policy
+AUTH0_CLIENT_ID=00000000-0000-0000-0000-000000000000
+AUTH0_DOMAIN=login.test.cezzis.com
+AUTH0_AUDIENCE=https://api.cezzis.com/cocktails
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AUTH0_AUDIENCE key should go before the AUTH0_CLIENT_ID key

(UnorderedKey)